src/queries/categories.js

```diff
@@ -7,6 +7,15 @@ export const GET_CATEGORIES = gql`
       name
       pointsOfInterestCount
       toursCount
+      children {
+        id
+        name
+        pointsOfInterestCount
+        toursCount
+      }
```

Comment:
in order to have multiple nested data in this request, we need more layers here:

```
children {
  id
  name
  pointsOfInterestCount
  toursCount
  children {
    id
    name
    pointsOfInterestCount
    toursCount
    children {
      id
      name
      pointsOfInterestCount
      toursCount
      children {
        id
        name
        pointsOfInterestCount
        toursCount
        children {
          id
          name
          pointsOfInterestCount
          toursCount
        }
      }
    }
  }
}
```

Reply:
for now we use the multi-nested query structure